build.psm1

```diff
@@ -1652,15 +1663,15 @@ function Start-TypeGen
 
     Push-Location "$PSScriptRoot/src/Microsoft.PowerShell.SDK"
     try {
-        $ps_inc_file = "$PSScriptRoot/src/TypeCatalogGen/powershell.inc"
+        $ps_inc_file = "$PSScriptRoot/src/TypeCatalogGen/$($incFileName)"
```

Comment:
`$($incFileName)` can be replaced by `$incFileName`

Reply:
In this case we can do that but what do you think about my comment above where this is not always possible?

Reply:
PowerShell support this syntax for variable: `${var name}`, so you can use `"${varname}_xxx.inc"` in that case.